GroupIS1024/Models/Player.cs

```diff
@@ -27,11 +28,12 @@ public class Player
 
         public string University { get; set; }
 
+        [DataType(DataType.Date)]
         public DateTime DateOfBirth { get; set; }
 
-        public Team Team { get; set; }
+        public Team? Team { get; set; }
```

Comment:
Given the previous models. It is impossible to create either team, player, or coach first. All are interdependent on each other to be created. So, I have made foreign key properties nullable to resolve the chicken and egg problem